webpack/scenes/BootedContainerImages/BootedContainerImagesPage.js

```diff
@@ -0,0 +1,242 @@
+              <Tr ouiaId="table-error">
+                <Td colSpan={100}>
+                  <EmptyPage message={{ type: 'error', text: errorMessage }} />
+                </Td>
+              </Tr>
+            )}
+          </Tbody>
+          {results?.map((result, rowIndex) => {
+            const { bootcBootedImage, digests } = result;
+            const isExpanded = imageIsExpanded(bootcBootedImage);
+            return (
+              <Tbody key="bootable-container-images-body" isExpanded={isExpanded}>
+                <Tr key={bootcBootedImage} ouiaId={`table-row-${rowIndex}`}>
+                  <>
+                    <Td
+                      expand={digests.length > 0 && {
+                        rowIndex,
+                        isExpanded,
+                        onToggle: (_event, _rInx, isOpen) =>
+                          expandedImages.onToggle(isOpen, bootcBootedImage),
+                        expandId: 'booted-containers-expander',
+                      }}
+                    />
+                    {columnNamesKeys.map(k => (
+                      <Td
+                        key={k}
+                        dataLabel={keysToColumnNames[k]}
+                      >
+                        {columns[k].wrapper ? columns[k].wrapper(result) : result[k]}
+                      </Td>
+                    ))}
+                  </>
+                </Tr>
+                {digests ?
+                  <Tr isExpanded={isExpanded} ouiaId={`table-row-outer-expandable-${rowIndex}`}>
+                    <Td />
+                    <Td colSpan={3}>
+                      <ExpandableRowContent>
+                        <TableComposable variant="compact" isStriped ouiaId={`table-composable-expanded-${rowIndex}`}>
+                          <Thead>
+                            <Tr ouiaId={`table-row-inner-expandable-${rowIndex}`}>
+                              <Th width={50}>{__('Image digest')}</Th>
+                              <Th width={50}>{__('Hosts')}</Th>
+                            </Tr>
+                          </Thead>
+                          <Tbody>
+                            {digests.map((digest, index) => (
+                              <Tr key={digest} ouiaId={`table-row-expandable-content-${index}`}>
+                                <Td>{digest.bootc_booted_digest}</Td>
+                                <Td>
+                                  <a href={`/hosts?search=bootc_booted_digest%20=%20${digest.bootc_booted_digest}`}>{digest.host_count}</a>
+                                </Td>
+                              </Tr>
+                            ))}
+                          </Tbody>
+                        </TableComposable>
+                      </ExpandableRowContent>
+                    </Td>
+                  </Tr> : null}
+              </Tbody>
+            );
+          })}
+        </TableComposable>
+        {results.length > 0 && !errorMessage && bottomPagination}
```

Comment:
More readable to not have `bottomPagination` and render the Pagination component directly here.

Reply:
Fixed!